Update to README to bring inline with logging repo (#100)
ministryofjustice/staff-device-dns-dhcp-infrastructure: documentation/azure_ad.md

# DNS / DHCP Azure AD SSO Provisioning
## Authentication with Azure AD
Azure AD provides the authorization backend, via [AWS Cognito](https://aws.amazon.com/cognito/), and is manually provisioned following the guide below.

## Caveats
- Due to the session time in Cognito, changes to users and / or roles for users 
whom have already signed into the admin portal can take up to around 15 minutes 
to propagate to the admin system. This is because updated sessions are only pulled from Azure once a Cognito session has timed out.

## Prerequisites

> - You will need access to a Devl MoJ account (ask in the Slack channel for this)

Reviewer: Might be worth changing this to: (The future owners may not be using slack)
```suggestion
- You will need access to the Devl MoJ Azure account.
```
